Marlin/SanityCheck.h

```diff
@@ -309,7 +309,7 @@
   #endif
 
   #if EXTRUDERS > 1 || ENABLED(HEATERS_PARALLEL)
-    #if !HAS_HEATER_1
+    #if (!HAS_HEATER_1 && DISABLED(PNEUMATICS))
```

Comment:
Can you leave a comment for why this is needed? Without a comment it feels weird that Pneumatics has anything to do with heating.